boilerplate.js

```diff
@@ -3,12 +3,28 @@ const fs    = require('fs')
     , after = require('after')
     , cpr   = require('cpr')
 
+function adjustForLang(orig, lang, cb) {
+  if (!lang) {
+    process.nextTick(function() { cb(orig); });
+    return;
+  }
+
+  var extName = path.extname(orig),
+      langAware = orig.slice(0, -extName.length) + '.' + lang + extName;
+
+  fs.exists(langAware, function(exists) {
+    cb(exists ? langAware : orig);
```

Comment:
`fs.exists` will be deprecated in future versions because it uses non-standard api. I think its better to stay standard in the callback, so it should be `cb(null, exists ? langAware : orig )`